components/TestLibraryStepper/TestLibraryStepper.tsx

```diff
@@ -3,8 +3,10 @@ import {Step, StepLabel, Stepper, Typography} from '@material-ui/core';
 import {MuiThemeProvider} from '@material-ui/core/styles';
 import {CandidateTestStatus} from "../../pages/api/candidateApiClient.module";
 import TestSwitchStepIcon from "../TestLibraryStepperIcons/TestLibraryStepperIcons";
-import {TestSwitchConnector, TestSwitchTheme} from "../TestLibraryOverrides/TestLibraryOverrides"
+import {h1Style, TestSwitchConnector, TestSwitchTheme} from "../TestLibraryOverrides/TestLibraryOverrides"
 import Link from "next/link";
+import scss from '../../pageStyles/index.module.scss';
```

Comment:
I'd be a little careful about importing classes from a different component's styles.

The concept of css modules (and React components) is that its a nice neat self contained thing.
The component includes only the stuff it needs, and the CSS module provides the styles for that component.

This is powerful and popular because everything is self contained - I can change whatever I like about that component and how it looks and be confident that I won't break anything else.

However, if we start importing styles that belong to a different component, then we break that model, and its no longer self contained.

Instead, I'd have my own CSS module for this component, and if I need to share styles, that's what mixins are for.

Reply:
ah!yes, that was not a great path to go down. Fixed and pushed now ✔️